src/hotspot/share/interpreter/interpreterRuntime.cpp

```diff
@@ -668,6 +668,7 @@ void InterpreterRuntime::resolve_get_put(Bytecodes::Code bytecode, int field_ind
   } // end JvmtiHideSingleStepping
 
   // check if link resolution caused cpCache to be updated
+  ConstantPoolCache* cache = pool->cache();
```

Comment:
Is this needed? I don't see `cache` being used within the function.

Reply:
I removed this line.